packages/adp-tooling/src/prompts/add-component-usages/index.ts

```diff
@@ -61,14 +61,23 @@ function validatePromptId(
         return validationResult;
     }
 
-    if (isCustomerBase && !hasCustomerPrefix(value)) {
-        return t('validators.errorInputInvalidValuePrefix', {
-            value: t('prompts.component.usageIdLabel'),
-            prefix: NamespacePrefix.CUSTOMER
-        });
+    if (isCustomerBase) {
+        if (!hasCustomerPrefix(value)) {
+            return t('validators.errorInputInvalidValuePrefix', {
```

Comment:
Duplicated code in add-new-model/index.ts. Can we extract the logic and reuse it in both places?